quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/integration/jaxrs/QuarkusKeycloakApplication.java

```diff
@@ -53,16 +67,21 @@ protected void loadConfig() {
 
     @Override
     public Set<Object> getSingletons() {
-        Set<Object> singletons = super.getSingletons().stream()
-                .filter(QuarkusKeycloakApplication::filterSingletons)
-                .collect(Collectors.toSet());
+        return Set.of();
```

Comment:
Can we remove the `ObjectMapperResolver` singleton as we already have the extended version `QuarkusObjectMapperResolver` in classes? Or should the singleton exist?

Reply:
This is how it is working already, isn't it? I mean, the `ObjectMapperResolver` should not be registered as an extension at runtime.